kalite/i18n/__init__.py

```diff
@@ -371,7 +371,7 @@ def update_jsi18n_file(code="en"):
     request.path = output_file
     request.session = {settings.LANGUAGE_COOKIE_NAME: code}
 
-    response = javascript_catalog(request, packages=('ka-lite.locale',))
+    response = javascript_catalog(request, packages=('ka-lite.locale',), domain="django")
```

Comment:
Why did the JS catalog become so big? Because we now have to include the entire `django` domain in order to translate as much exercises as possible, the `djangojs` domain is almost a subset of `django`. This means the JS catalog file would be really big, but see my workaround to make sure it's only downloaded once per user.